### ISSUE TITLE
Image Save Location

### ISSUE THREAD

**Issue by ArcaneTSGK** [cofoundry-cms/cofoundry#316]
We currently have an instance of our CMS deployed to AWS EC2 via CodeDeploy, during this process the application is removed from the server with the latest version of the application being deployed.

This of course removes everything under App_Data, meaning all image files on the server are lost on every deployment. 

Is there a way to configure the image upload in CoFoundry to save the images to another location, ideally this would be a separate drive on the server, or at the very least a directory outside of the application?

**Comment by HeyJoel** [comment]
There's two ways you could resolve this, one is to change the [`Cofoundry:FileSystemFileStorage:FileRoot` setting](https://www.cofoundry.org/docs/references/common-config-settings#filesystemfilestoragesettings) e.g.

**appsettings.json**
```js
{
  "Cofoundry": {
    "FileSystemFileStorage:FileRoot": "F:\\Files\\MySite",
  }
}
```

The other method would be to use a plugin like [Cofoundry.Plugins.Azure](https://github.com/cofoundry-cms/Cofoundry.Plugins.Azure) if you were using Azure, which would allow you to use blob storage. Unfortunately there's no S3 equivalent plugin at the moment, but I think it would be fairly easy to create one if you wanted to go down that route, all you'd need to do is create an [`IFileStoreService`](https://github.com/cofoundry-cms/Cofoundry.Plugins.Azure/blob/master/src/Cofoundry.Plugins.Azure/AzureBlobFileService.cs) implementation and [register it to override the base implementation](https://github.com/cofoundry-cms/Cofoundry.Plugins.Azure/blob/master/src/Cofoundry.Plugins.Azure/AzureDependencyRegistration.cs)

We should add some documentation on this either in the [deployment docs](https://www.cofoundry.org/docs/getting-started/publishing-and-deployment) or the [assets docs](https://www.cofoundry.org/docs/content-management/images)

**Comment by ArcaneTSGK** [comment]
Thank you for your quick response, would you like me to leave this issue open for now as a reminder to add documentation?

Cheers!

**Comment by HeyJoel** [comment]
Yes please, I think we need to update the docs.

**Comment by HeyJoel** [comment]
Documentation complete, will be published with the v0.12 released.
